pkg/kv/kvserver/store_snapshot.go

```diff
@@ -374,7 +376,9 @@ func (kvSS *kvBatchSnapshotStrategy) sendBatch(
 }
 
 // Status implements the snapshotStrategy interface.
-func (kvSS *kvBatchSnapshotStrategy) Status() string { return kvSS.status }
+func (kvSS *kvBatchSnapshotStrategy) Status() redact.SafeString {
+	return redact.SafeString(kvSS.status)
```

Comment:
would be cleaner to make the `status` field a `redact.SafeString`, no?

Reply:
True, done.